src/routes/Products/components/BasicInfo/index.jsx

```diff
@@ -217,6 +218,8 @@ const BasicInfo = ({
     }, 100);
   };
 
+  console.log(breadcrumb, "breadcrumb rece");
```

Comment:
let's remove this console log and run eslint and tslint for good measure

Reply:
Ah sorry! will update the PR.

Reply:
@brooketopcoder updated the code. Running tslint has introduced changes in other files as well. is that alright?